src/test/java/automation/pages/LoginPage.java

```diff
@@ -1,13 +1,13 @@
-package login_page_test.pages;
+package automation.pages;
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 
-public class LoginPage { // todo final
-    private WebDriver driver;
-    private By usernameLocator = By.id("page_x002e_components_x002e_slingshot-login_x0023_default-username");
-    private By passwordLocator = By.id("page_x002e_components_x002e_slingshot-login_x0023_default-password");
-    private By loginButtonLocator = By.id("page_x002e_components_x002e_slingshot-login_x0023_default-submit-button");
+public class LoginPage {
+    private final WebDriver driver;
+    private final By usernameLocator = By.id("page_x002e_components_x002e_slingshot-login_x0023_default-username");
```

Comment:
Why  in some cases you initialize locators in Page Object constructor, in other you do it in class declaration? Do it the same way everywhere. 
The most common way for today is this: https://www.ontestautomation.com/findby-strategies-for-selenium-explained/

Reply:
I can't find the place when I'm initializing locator via constructor. In one class I'm doing it trough
method. In this one class I want to initialize locator with group name which are passed from test case. Each time group name can be different that's why I'm not initializing locator in class declaration. 
But I refactored code using @FindBy annotation in the rest PageObject classes.